core/common/src/main/java/alluxio/ConfigurationValueOptions.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+/**
+ * Options for getting configuration values.
+ */
+public final class ConfigurationValueOptions {
+  private boolean mUseDisplayValue = false;
+  private boolean mUseRawValue = false;
+  private boolean mCheckNullValue = false;
```

Comment:
this makes it difficult to determine whether `get(PropertyKey key, ConfigurationValueOptions)` is `Nullable` or not, causing issues for static analysis tools